pocs/filterwheel/filterwheel.py

```diff
@@ -0,0 +1,224 @@
+import threading
+
+from astropy import units as u
+
+from pocs.base import PanBase
+from pocs.utils import listify
+
+
+class AbstractFilterWheel(PanBase):
+    """
+    Base class for all filter wheels
+
+    Args:
+        name (str, optional): name of the filter wheel
+        model (str, optional): model of the filter wheel
+        camera (pocs.camera.*.Camera, optional): camera that this filter wheel is associated with.
+        filter_names (list of str): names of the filters installed at each filter wheel position
```

Comment:
I can imagine a scenario where I want to pass a `dict` of names/positions but might not be necessary.

Reply:
I'm not seeing a need right now. Can add that option later, if it does turn out to be necessary.